elements/pl-dropdown/pl-dropdown.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
If we're not using this file lets remove it, same with the `pl-dropdown.js`.

Reply:
Done here: 5194abd93082dd78015aad237589c549ea26fdcd